packages/console/src/components/Tooltip/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { Nullable } from '@silverhand/essentials';
+import classNames from 'classnames';
+import React, { ReactNode, RefObject, useEffect, useState } from 'react';
+import { createPortal } from 'react-dom';
+
+import * as styles from './index.module.scss';
+
+type Props<T> = {
+  content: ReactNode;
+  domRef: RefObject<Nullable<T>>;
+  className?: string;
```

Comment:
nit: Consider adding a delay timeout (by default 2000ms maybe?) before showing up the tooltip

Reply:
don't think we need it for now - we can leave this issue to Guamian